tests/test_table_reset.py

```diff
@@ -0,0 +1,28 @@
+from sfbase import SFBaseTest
+from tap_tester.base_suite_tests.table_reset_test import TableResetTest
+
+
+class SFTableResetTest(TableResetTest, SFBaseTest):
+    """tap-salesforce Table reset test implementation"""
+
+    salesforce_api = 'BULK'
```

Comment:
This should probably go in `SFBaseTest` since it's used by `SFBaseTest.get_properties`

Reply:
We should probably set a default in the base case, which can be overriden by setting the property here if needed.  Also, do we need to run this test for both BULK and REST?  We could import TableResetTest again in another module and class and set it in that test to REST.

Reply:
moved this to sfbase.py